cmd/makenaclboxkey/main.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"crypto/rand"
+	"crypto/sha256"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"golang.org/x/crypto/nacl/box"
+)
+
+// KeyPair is a public and private key usable with NACL BOX
+type KeyPair struct {
+	PublicKey, PrivateKey string
+}
+
+func main() {
```

Comment:
Why not have this as a subcommand under the `sops` binary? Could be `sops naclbox-gen` or similar?

Reply:
Agreed, but I'd prefer to keep the longer `makenaclboxkey` name.